Index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="Style/main.css">
+    <link rel="stylesheet" href="Style/homePageStyle.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <title>Bokolo Drill</title>
+</head>
+<body>
+  <div class="center">
+    <span></span><span></span><span></span>
+  </div>
+    <header class="header-container">
+      <div class="circle"></div>
+      <nav>
+        <ul class="first-navbar">
+          <li><i class="bi bi-facebook"></i></li>
+          <li><i class="bi bi-twitter"></i></li>
+          <li><a>English</a></li>
+          <li><a>My account</a></li>
+          <li><a>logout</a></li>
+        </ul>
+      </nav>
+        <nav class="navbar">
+          <a href="Index.html"><img class="brand-logo" src="Images/logo_image/logo-image.png" alt="Brand-name-and-logo"></a>
+         <ul class="nav-list-items">
+          <li class="list-item"><a  href="Index.html">Home</a></li>
+          <li class="list-item"><a  href="Interfaces/About.html">About</a></li>
+          <li class="list-item"><a  href="#">Program</a></li>
+          <li class="list-item"><a  href="#">Sponsor</a></li>
+          <li class="list-item"><a  href="#">New</a></li>
+          <li class="list-item"><a  href="#">Ticket</a></li>
+         </ul>
+         <div class="hambuger-container">
+         <div class="bar"></div>
+         <div class="bar"></div>
+         <div class="bar"></div>
+         </div>
+        </nav>
+      </header>
+      <section class="landing-section">
+        <div class="intro-section">
+          <h1 class="openingText">
+            Quarter mboko rappers           
+          </h1>
+          
+               <h2 class="captureText">
+                Presenting! Mboko gods among us 2022
+               </h2>
+          
+
+               <p class="explaining-text">
+                Come one come all. Introducing the best covered up mboko divine beings and goddesses among us this year in December. Get ready to be stunned by these incredible rappers.
+               </p>
+
+               <p class="eventDate">2022.12.10(SAT) ~ 11(SUN)</p>
+               <span class="venue">At Congrate hall Bamenda Cameroon</span>
+        </div>
+      </section>
+
+      <section class="shedule-section">
+        <div class="text-time">
+          <h2 class="text-header">Main Program</h2>
+          <hr class="line">
+        </div>
+        
+        <div class="shedule-container">
```

Comment:
Kindly, I suggest you use the semantic tag "ARTICLE", this is ideal for repetitive content like this